uaclient/cli/vulnerability/list.py

```diff
@@ -0,0 +1,460 @@
+    msgs = []
+    sorted_priority_count = sorted(
+        count_by_priority.items(),
+        key=lambda item: vuln_util.CVE_PRIORITIES.index(item[0]),
+    )
+
+    for priority, count in sorted_priority_count:
+        if count:
+            msgs.append(
+                "{} {}".format(
+                    count, vuln_util.get_priority_with_color(priority)
+                )
+            )
+
+    return ", ".join(msgs)
+
+
+def _create_fixable_cves_count(vulnerabilities) -> str:
+    msg = messages.CLI_VULNERABILITY_LIST_FIXES_AVAILABLE_HEADER + "\n"
+    fixable_vulnerabilities_info = _get_info_from_vulnerabilities(
+        vulnerabilities, fixable_vuln=True
+    )
+    count_with_color = _get_fixable_color_count(fixable_vulnerabilities_info)
+    msg += (
+        "    "
+        + count_with_color
+        + " "
+        + messages.CLI_VULNERABILITY_LIST_FIXES_AVAILABLE_COUNT
+        + " ("
+        + _get_count_msg_by_priority(
+            fixable_vulnerabilities_info.count_by_priority
+        )
+        + ")"
+        + "\n"
+    )
+
+    return msg
+
+
+def _create_fixable_usns_count(vulnerabilities):
+    msg = messages.CLI_VULNERABILITY_LIST_FIXES_AVAILABLE_HEADER + "\n"
+    fixable_vulnerabilities_info = _get_info_from_vulnerabilities(
+        vulnerabilities, fixable_vuln=True
+    )
+    msg += (
+        "    "
+        + str(fixable_vulnerabilities_info.count)
+        + " "
+        + messages.CLI_VULNERABILITY_LIST_FIXES_AVAILABLE_COUNT
+        + "\n"
+    )
+    return msg
+
+
+def _create_unfixable_cves_count(vulnerabilities) -> str:
+    msg = messages.CLI_VULNERABILITY_LIST_UNFIXABLE_AVAILABLE_HEADER + "\n"
+    unfixable_vulnerabilities_info = _get_info_from_vulnerabilities(
+        vulnerabilities, fixable_vuln=False
+    )
+    msg += (
+        "    "
+        + str(unfixable_vulnerabilities_info.count)
+        + " "
+        + messages.CLI_VULNERABILITY_LIST_UNFIXABLE_AVAILABLE_COUNT
+        + " ("
+        + _get_count_msg_by_priority(
+            unfixable_vulnerabilities_info.count_by_priority
+        )
+        + ")"
+        + "\n"
+    )
+
+    return msg
+
+
+def _create_unfixable_usns_count(vulnerabilities):
+    msg = messages.CLI_VULNERABILITY_LIST_UNFIXABLE_AVAILABLE_HEADER + "\n"
+    unfixable_vulnerabilities_info = _get_info_from_vulnerabilities(
+        vulnerabilities, fixable_vuln=False
+    )
+    msg += (
+        "    "
+        + str(unfixable_vulnerabilities_info.count)
+        + " "
+        + messages.CLI_VULNERABILITY_LIST_UNFIXABLE_AVAILABLE_COUNT
+        + "\n"
+    )
+
+
+def _create_already_fixed_cves_count(applied_fixes_count):
+    msg = ""
+    if any(count for pocket, count in applied_fixes_count["count"].items()):
+        msg += messages.CLI_VULNERABILITY_LIST_FIXES_APPLIED_HEADER + "\n"
+        for pocket, count in applied_fixes_count["count"].items():
+            if not count:
+                continue
+
+            msg += (
+                "    "
```

Comment:
I know this was implemented before I sent the `Block` structure, but this should be a Block so we indent it correctly
```
Vulnerabilities with applied fixes:
    2079 applied via Ubuntu Security (2 critical, 38 high, 1399 medium, 604 low,
 36 negligible)
```

(and then you don't need to add color+spaces to it)